packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 * Added
   * Added test to cover enabling zebra striping for the `terra-table` component.
 
+* Updated
```

Comment:
Same as above - probably don't need a changelog entry.

Reply:
Per discussion with the team the changelog entry for the terra-framework-docs should stay